src/lib/journey/callbacks/device-profile/device-profile.mock.ts

```diff
@@ -1,23 +1,59 @@
-export const docsExample = {
-  type: 'DeviceProfileCallback',
-  output: [
+import type { Callback, CallbackType } from '@forgerock/javascript-sdk';
+
+export const deviceProfileMockNoMessage = {
+  callbacks: [
     {
-      name: 'metadata',
-      value: true,
-    },
-    {
-      name: 'location',
-      value: true,
-    },
-    {
-      name: 'message',
-      value: 'Collecting.....',
+      type: 'DeviceProfileCallback' as CallbackType,
+      output: [
+        {
+          name: 'metadata',
+          value: true,
+        },
+        {
+          name: 'location',
+          value: true,
+        },
+        {
+          name: 'message',
+          value: '',
+        },
+      ],
+      input: [
+        {
+          name: 'IDToken1',
+          value:
```

Comment:
Let's remove these values here and the one below as they won't come pre-populated like this.